internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/service/QualysScanner.java

```diff
@@ -207,6 +212,17 @@ private Boolean isValidParameters(ScannerScanRequest scannerScanRequest) throws
             scanContext.setProgressiveScanning(parameterMap.get(QualysScannerConstants.PROGRESSIVE_SCAN).get(0));
         }
 
+        // Validate report template ID
+        if (StringUtils.isEmpty(parameterMap.get(QualysScannerConstants.REPORT_TEMPLATE_ID).get(0))) {
+            scanContext.setReportTemplateId(QualysScannerConfiguration.getInstance().getDefaultReportTemplateID());
+            String logMessage =
+                    "Report template ID for the scan is not provided. Default report template ID" + " is set for "
```

Comment:
Did we miss something here. After the "+"

Reply:
Resolved